<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitResultsWindow</class>
 <widget class="QMainWindow" name="FitResultsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="resultsTab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabFitResults">
       <attribute name="title">
        <string>Fit Results</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </widget>
      <widget class="QWidget" name="tabFitParams">
       <attribute name="title">
        <string>Fit Params</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabMcResults">
       <attribute name="title">
        <string>Mc Resuts</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabPlotData">
       <attribute name="title">
        <string>Plot Data</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabStatistic">
       <attribute name="title">
        <string>Statistic</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="textStatistic"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_File"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_File"/>
  </widget>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/menu/Icons/ShowGEMResultsIcon.png</normaloff>:/menu/Icons/ShowGEMResultsIcon.png</iconset>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="text">
    <string>Save File...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../fgui.qrc"/>
 </resources>
 <connections/>
</ui>
